rust/src/go/nmstate/nmstate.go

```diff
@@ -0,0 +1,111 @@
+package nmstate
```

Comment:
Add the CGO build tag so this is compiled only whent CGO is enabled.

```
// +build cgo
```

Reply:
Done, thanks!